src/main/java/uk/gov/hmcts/et/common/model/hmc/ValidationError.java

```diff
@@ -55,7 +55,7 @@ private ValidationError() {
     public static final String PARTY_DETAILS_NULL_EMPTY = "Party id can not be empty";
     public static final String PARTY_DETAILS_MAX_LENGTH = "Party id must not be more than 40 " + CHARACTERS_LONG;
     public static final String PARTY_ROLE_MAX_LENGTH = "Party role must not be more than 40 " + CHARACTERS_LONG;
-    public static final String NAME_NULL_EMPTY = "Name can not be empty";
+    public static final String NAME_NULL_EMPTY = "Party name can not be empty";
```

Comment:
Is this only called in one place? It might be worth renaming this to `PARTY_NAME_NULL_EMPTY`. You wouldn't expect a var called `NAME_NULL_EMPTY` to be specific to "party". Makes it less reusable

Reply:
Have made a commit to resolve this thank you